cpu/sam0_common/Makefile

```diff
@@ -1,3 +1,7 @@
 DIRS = periph
 
+ifneq (, $(filter sam0_eth, $(USEMODULE)))
+    DIRS += sam0-eth
```

Comment:
maybe better u derscore for consistency with the module name?

Reply:
Done